bosh-stemcell/lib/bosh/stemcell/build_environment.rb

```diff
@@ -181,9 +181,12 @@ def exclude_exclusions
       when 'google'
         ' --tag ~exclude_on_google'
       else
-        ''
+        nil
       end,
-      ].join(' ').rstrip
+      unless operating_system.variant.nil?
```

Comment:
My small caveman brain has trouble parsing this. Is this the same as `if operating_system.variant`?

Functionally it is different because `false` values would behave differently in the two, but that's probably not a reasonably valid value for `variant` I'm guessing

Reply:
Yeah, I guess you are right, getting rid of some of the explicitness in order to improve readability would make the code more idiomatic.